pkg/settings/cresettings/settings.go

```diff
@@ -128,6 +144,12 @@ type Schema struct {
 	WorkflowTriggerRateLimit        Setting[config.Rate]
 	GatewayIncomingPayloadSizeLimit Setting[config.Size]
 
+	VaultCiphertextSizeLimit          Setting[config.Size]
+	VaultIdentifierKeySizeLimit       Setting[config.Size] `unit:"{byte}"`
+	VaultIdentifierOwnerSizeLimit     Setting[config.Size] `unit:"{byte}"`
+	VaultIdentifierNamespaceSizeLimit Setting[config.Size] `unit:"{byte}"`
```

Comment:
You can omit unit from these. It is applied automatically by the `Size` func.
```suggestion
	VaultIdentifierKeySizeLimit       Setting[config.Size]
	VaultIdentifierOwnerSizeLimit     Setting[config.Size]
	VaultIdentifierNamespaceSizeLimit Setting[config.Size]
```